specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessments.json

```diff
@@ -261,13 +261,29 @@
         },
         "links": {
           "$ref": "#/definitions/AssessmentLinks"
+        },
+        "metadata": {
+          "$ref": "./assessmentMetadata.json#/definitions/SecurityAssessmentMetadataProperties"
+        },
+        "partnersData": {
+          "$ref": "#/definitions/SecurityAssessmentPartnerData"
         }
       },
       "required": [
         "resourceDetails",
         "status"
       ]
     },
+    "SecurityAssessmentPartnerData": {
+      "type": "object",
+      "description": "Data regarding 3rd party partner integration",
+      "properties": {
+        "secret": {
```

Comment:
>secret [](start = 9, length = 6)

We now have a swagger extension to annotate secrets with. Can you add the following to this property?
```
"x-ms-secret": true
```